client/components/mma/billing/Billing.stories.tsx

```diff
@@ -35,22 +35,34 @@ export const NoSubscription: StoryObj<typeof Billing> = {
 
 	parameters: {
 		msw: [
-			rest.get('/api/me/mma', (_req, res, ctx) => {
-				return res(ctx.json(toMembersDataApiResponse()));
+			http.get('/api/me/mma', () => {
+				return new Response(JSON.stringify(toMembersDataApiResponse()), {
+					headers: {
+						'Content-Type': 'application/json',
+					},
+				})
```

Comment:
could we use `HttpResponse` here as well like you have in contactUs.stories.tsx:

```
http.get('/api/me/mma', () => {
  return HttpResponse.json(toMembersDataApiResponse())
})
```